uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/HIIConfigAccess/BlackBoxTest/HIIConfigAccessBBTestMain.c

```diff
@@ -381,6 +381,71 @@ GetDevicePath (
   return Status;
 }
 
+EFI_STATUS
+GetDevicePathProtocol (

```

Comment:
Why don't we make a similar change to HiiConfigAccess protocol test? having a function called LocateDevicePathFromHIIConfigAccess ()